### PR TITLE
Add visualize method to MultiTableMetadata class

### LINKED ISSUE #884: Add visualize method to MultiTableMetadata class
### Problem Description

_As a user, it would be helpful to visualize multi-table metadata and see if all the relationships and keys are correct._

### Expected behavior

- Add a `visualize` method
- We can largely borrow from the existing [visualization.visualize method](https://github.com/sdv-dev/SDV/blob/d90ed40ef5e8daeca636fdf07caf27e42c13d09a/sdv/metadata/visualization.py#L108) however will need to slightly modify it
- Parameters:
    - `show_table_details`: boolean that decides what details to show. When True, lists the column names and primary/foreign keys in each table,. When False, only lists the table name.
    - `show_relationship_labels`: boolean that toggles whether to label the edges with column names. When True: Labels every edge with the column names <child column> → <parent column> eg. purchaser_id → user_id.
    - `output_filepath`: string that specifies where to save the visualization. Defaults to `None`. If `None`, don't save anything.
- The method should always return a `graphviz.Digraph` object
- Errors:
    - If the file already exists raise the following error
    `Error: This file already exists. Specify a new file name.`
    - If the filetype is unsupported, raise the following error
    `Error: Unable to save a visualization with this file type. Try a supported file type like 'png', 'jpg' or 'pdf'. For a full list, see 'https://graphviz.org/docs/outputs/'`
    - If the filetype isn't in the path, raise the following error
    `Error: No file type specified. The filepath must end in a supported type such as 'png', 'jpg' or 'pdf'. For a full list, see 'https://graphviz.org/docs/outputs/'`

```python
>>> digraph = metadata.visualize()

>>> digraph = metadata.visualize(
      output_filepath='my_schema.png',
      show_table_details=False,
      show_relationship_details=False)

# Error cases
>>> digraph = metadata.visualize(output_filepath='my_schema.png')
Error: This file already exists. Specify a new file name.

>>> digraph = metadata.visualize(output_filepath='my_schema.docx')
Error: Unable to save a visualization with this file type. Try a supported file type like 'png', 'jpg' or 'pdf'. For a full list, see 'https://graphviz.org/docs/outputs/'

>>> digraph = metadata.visualize(output_filepath='my_schema')
Error: No file type specified. The filepath must end in a supported type such as 'png', 'jpg' or 'pdf'. For a full list, see 'https://graphviz.org/docs/outputs/'
```

### PR DESCRIPTION
**Pull request by amontanez24**
Resolves #884

### CODE REVIEW

File: tests/unit/metadata/test_multi_table.py [hunk 1]
```diff
@@ -235,3 +235,203 @@ def test___repr__(self, mock_json):
         # Assert
         mock_json.dumps.assert_called_once_with(instance.to_dict(), indent=4)
         assert res == mock_json.dumps.return_value
+
+    def get_metadata(self):
+        """Set the tables and relationships for metadata"""
```
Comment (fealho) [inline tests/unit/metadata/test_multi_table.py:240]: . at the end

**Review by fealho — APPROVED** [review]
Looks good 👍

File: sdv/metadata/multi_table.py [hunk 2]
```diff
@@ -35,6 +36,65 @@ def _set_metadata_dict(self, metadata):
         for relationship in metadata.get('relationships', []):
             self._relationships.append(relationship)
 
+    def visualize(self, show_table_details, show_relationship_labels, output_filepath=None):
```
Comment (pvk-developer) [inline sdv/metadata/multi_table.py:39]: I think that `show_table_dateils` and `show_relationship_labels` should default to some value as we want to be able to run `digraph = metadata.visualize()` .

Comment (pvk-developer) [inline sdv/metadata/multi_table.py:39]: Should `show_table_details` and `show_relationship_labels` be optional, and be able to run `digraph = metadata.visualize()` ?

**Review by pvk-developer — CHANGES_REQUESTED** [review]

**Review by pvk-developer — APPROVED** [review]

**Review by fealho — APPROVED** [review]

**Review by pvk-developer — APPROVED** [review]

**Pull request merged**